### ISSUE TITLE
Let's include an option to activate/deactivate printing to the console

### ISSUE THREAD

**Issue by j-hartmann** [JasperLS/Understanding_Videos_at_Scale#1]
@JasperLS

**Comment by bell-xiong** [comment]
when i run this code, the fold of images is none, and the traceback shows that "TypeError: 'tuple' object does not support item assignment", so ,could you please answer my question?

**Comment by JasperLS** [comment]
Yes, there seem to be 2 separate issues, one with scenedetect and one with object dection. I will fix it during the next days.

**Comment by bell-xiong** [comment]
thank you for your reply!
looking forward to your update.
and i also want to know can the code run in&nbsp; python?&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JasperLS/Understanding_Videos_at_Scale"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月25日(星期一) 晚上8:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [JasperLS/Understanding_Videos_at_Scale] Let's include an option to activate/deactivate printing to the console (#1)





 
Yes, there seem to be 2 separate issues, one with scenedetect and one with object dection. I will fix it during the next days.
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JasperLS** [comment]
yes, the code can be transferred to your local machine and be run in Py - however, it may require custom enviroment set-ups, and you need to download all required files (as they are downloaded in our script).

By the way, the scene detect issue is fixed. All options except for the object detection should work now. I will continue to fix the object detection asap.

**Comment by bell-xiong** [comment]
thank you for your response.
I run the code on my local machine. I'm able to obtain all of the results except the visual variance result.I'm not sure if it's a problem with my modified code.&nbsp;Is it possible for me to send you my modified code and ask you to assist me in troubleshooting the issue?&nbsp;
&nbsp;Thank&nbsp;you&nbsp;for&nbsp;everything!
&nbsp;Enjoy&nbsp;yourself!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JasperLS/Understanding_Videos_at_Scale"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月8日(星期一) 晚上11:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [JasperLS/Understanding_Videos_at_Scale] Let's include an option to activate/deactivate printing to the console (#1)





 
yes, the code can be transferred to your local machine and be run in Py - however, it may require custom enviroment set-ups, and you need to download all required files (as they are downloaded in our script).
 
By the way, the scene detect issue is fixed. All options except for the object detection should work now. I will continue to fix the object detection asap.
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JuRaKlWi** [comment]
Had the same issue. Updating the scenedetect package to a version >0.6.0 fixed it for me

**Comment by conkeur** [comment]
> Had the same issue. Updating the scenedetect package to a version >0.6.0 fixed it for me

fixed it for me too.
